### PR TITLE
fix: removing Separated ingress

### PR DESCRIPTION
**Pull request by hamza-m-masood**
## Description

closes: https://github.com/camunda/camunda-docs/issues/4507
I am removing separated ingress from 8.7 docs

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by conceptualshark** [comment]
@hamza-m-masood Would you like me to help with the 8.7 docs review for updating where we use separated Ingress? Happy to take that on as part of this PR.

**Comment by conceptualshark** [comment]
@aabouzaid @hamza-m-masood I've gone ahead and found the other remaining instances of the separate ingress option in the 8.7 docs; this should be good to review again.

File: docs/self-managed/setup/install.md [hunk 1]
```diff
@@ -394,7 +394,7 @@ console:
 For more details, check [Console Helm values](https://artifacthub.io/packages/helm/camunda/camunda-platform#console-parameters).
 
 :::note
-Console Self-Managed requires the Identity component to authenticate. Camunda Helm Chart installs Identity by default. When logging in to Console when using port-forward, port-forward Keycloak service `kubectl port-forward svc/<RELEASE-NAME>-keycloak 18080:80` or configure Identity with Ingress as described in [combined and separated Ingress setup](/self-managed/setup/guides/ingress-setup.md).
+Console Self-Managed requires the Identity component to authenticate. Camunda Helm Chart installs Identity by default. When logging in to Console when using port-forward, port-forward Keycloak service `kubectl port-forward svc/<RELEASE-NAME>-keycloak 18080:80` or configure Identity with Ingress as described in the [Ingress setup guide](/self-managed/setup/guides/ingress-setup.md).
```
Comment (mesellings) [inline docs/self-managed/setup/install.md:397]:
```suggestion
Console Self-Managed requires the Identity component to authenticate. Camunda Helm Chart installs Identity by default. When logging in to Console when using port-forward, port-forward the Keycloak service `kubectl port-forward svc/<RELEASE-NAME>-keycloak 18080:80` or configure Identity with Ingress as described in the [Ingress setup guide](/self-managed/setup/guides/ingress-setup.md).
```

File: docs/self-managed/setup/guides/ingress-setup.md [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
-
-1. **Combined setup:** In this setup, there are two Ingress objects: one Ingress object for all Camunda 8 web applications using a single domain. Each application has a sub-path e.g. `camunda.example.com/operate`, and `camunda.example.com/optimize` and another Ingress which uses gRPC protocol for Zeebe Gateway e.g. `zeebe.camunda.example.com`.
-2. **Separated setup:** In this setup, each component has its own Ingress/host e.g. `operate.camunda.example.com`, `optimize.camunda.example.com`, `zeebe.camunda.example.com`, etc.
-
-There are no significant differences between the two setups. Rather, they both offer flexibility for different workflows.
+Camunda 8 Self-Managed has multiple web applications and gRPC services. Both can be accessed externally using Ingress with a **combined setup.** In this configuration, there are two Ingress objects: one Ingress object for all Camunda 8 web applications using a single domain. Each application has a sub-path e.g. `camunda.example.com/operate`, and `camunda.example.com/optimize` and another Ingress which uses gRPC protocol for Zeebe Gateway e.g. `zeebe.camunda.example.com`.
```
Comment (mesellings) [inline docs/self-managed/setup/guides/ingress-setup.md:14]:
```suggestion
Camunda 8 Self-Managed has multiple web applications and gRPC services. Both can be accessed externally using Ingress with a **combined setup.** In this configuration, there are two Ingress objects: one Ingress object for all Camunda 8 web applications using a single domain. Each application has a sub-path, for example `camunda.example.com/operate`, and `camunda.example.com/optimize`, and another Ingress that uses gRPC protocol for Zeebe Gateway, for example `zeebe.camunda.example.com`.
```
Suggestions for removing "e.g"

File: docs/self-managed/setup/guides/ingress-setup.md [hunk 3]
```diff
@@ -1,22 +1,17 @@
...
+title: "Ingress setup"
+description: "Camunda 8 Self-Managed Ingress setup and example configuration."
 ---
 
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
 :::caution
-The separated Ingress configuration has been deprecated in version 8.6. To ensure a smooth upgrade experience for new installations, we recommend using the **combined Ingress setup**.
+The separated Ingress configuration has been removed in Camunda version 8.7. This guide refers to only the **combined Ingress setup**.
```
Comment (mesellings) [inline docs/self-managed/setup/guides/ingress-setup.md:11]:
```suggestion
The separated Ingress configuration has been removed in Camunda version 8.7. This guide covers a **combined Ingress setup**.
```
Rewording suggestion

File: docs/self-managed/setup/deploy/local/local-kubernetes-cluster.md [hunk 4]
```diff
@@ -109,9 +109,7 @@ To get a full list of the deployed Camunda components and their network properti
 
 <TabItem value="ingress">
   
-Camunda 8 Self-Managed has multiple web applications and gRPC services. Both can be accessed using Kubernetes Ingress.
-
-In this example, we will use a combined Ingress configuration. For more information, refer to [combined and separated Ingress setup](/self-managed/setup/guides/ingress-setup.md).
+Camunda 8 Self-Managed has multiple web applications and gRPC services. Both can be accessed using Kubernetes Ingress. For more information, refer to the [Ingress setup guide](/self-managed/setup/guides/ingress-setup.md).
```
Comment (mesellings) [inline docs/self-managed/setup/deploy/local/local-kubernetes-cluster.md:112]:
```suggestion
Camunda 8 Self-Managed has multiple web applications and gRPC services. These can be accessed using Kubernetes Ingress. For more information, refer to the [Ingress setup guide](/self-managed/setup/guides/ingress-setup.md).
```
Rewording suggestion

**Review by mesellings — APPROVED** [review]
Lgtm 👍 , just some rewording suggestions, particularly to change "e.g." to "for example" as we avoid using "e.g." in the docs.

**Comment by conceptualshark** [comment]
@aabouzaid thank you for the optimize catch - this should be good to go now! 👍

**Review by mesellings — APPROVED** [review]
Lgtm 👍 🚀

**Comment by mesellings** [comment]
@hamza-m-masood Would you like us to merge and release this?

**Review by aabouzaid — APPROVED** [review]

**Comment by hamza-m-masood** [comment]
Thanks for the help everyone!

**Pull request merged**